flow/nlu/get_intents.yaml

```diff
@@ -1,45 +1,32 @@
 steps:
   - type: component.nlu.tokenize_text
-    text: (@ flow.result )
+    text: (@ flow.text )
```

Comment:
This flow seems unused. Is that right?

Also it is very similar to inbound. You could reuse that code passing a variable is_inbound...

Reply:
👍 I didn't finish the email flows this time (this is related, maybe a bug that it's unused). We can clean it up in the next issue: https://meyaai.atlassian.net/browse/MC-160?focusedCommentId=11835